src/style.css

```
@@ -1,3 +1,68 @@
+@import url("https://fonts.googleapis.com/css2?family=Montserrat&family=Roboto:wght@500&display=swap");
+
 body {
   margin: 0;
+  padding: 0;
+
+  font-family: "Roboto", sans-serif;
+  --color-decor: #00acdc;
+}
+
+.header {
+  margin: 0;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+}
+
+.nav :not(:first-child) {
```

```suggestion
.nav__item:not(:first-child) {
```
usem nav__item instead